src/main/resources/assets/nucleus/messages.properties

```diff
@@ -334,6 +334,19 @@ config.gamemode.separate=If true, then changing gamemode using the /gm command r
 config.protection.disablecrop=Disables crop trampling.
 config.protection.mobgriefing.flag=If true, mob griefing will be disabled except for the entities listed in the whitelist (which should be done by mob ID).
 
+config.scheduled.task.usewarning=Whether or not this has some warnings before the action.
+config.scheduled.task.warningtimelist=A list of times to warn before the action, doesn't have to be in order.
```

Comment:
Anywhere you use a `'` in the properties files, please double it up (like `''`), because it might otherwise player havoc with the formatting functions!